src/components/Filter/FilterGroup.js

```diff
@@ -3,24 +3,43 @@ import FilterItem from './FilterItem';
 import {capitalize} from '../../util';
 import styles from './FilterGroup.scss';
 import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import {getAvailableLanguages} from '../../util';
+
+const languages = getAvailableLanguages();
+
+/*
+* Checks if a tagItem is a valid language tag or if the language tag is 'undefined language'
+* Returns undefined if not.
+*/
+const getLanguageTagItemName = (tagItem) => {
+  return languages[tagItem] ? languages[tagItem] : tagItem === 'undefined language' ? tagItem : undefined;
+};
+
+/*
+* Extend this function when more groupName translations are added
+*/
+const translateGroupName = (groupName, t) => {
+  return groupName === 'language' ? t('filter.language') : groupName;
```

Comment:
Gruppene generelt for filteret skal være forhåndsdefinert i oppgaverepoet, i tillegg til captions for de. Men spårkdelen til filteret kan ha en fil i src/constants som f.eks. heter filterlanguages.js. Språk i filteret skal stå på den måten det skrives i det landet, språket blir brukt til. F.eks. hvis man har siden på engelsk, skal det fortsatt stå Norsk bokmål i filteret under Language-gruppen. Så filtergruppen for språk er det eneste du trenger å tenke på som skal oversettes.